setup.py

```diff
@@ -50,6 +54,15 @@
     "graphviz",  # used in plot tests
     "pytest-assume",  # sql counter check
     "decorator",  # sql counter check
+    "protoc-wheel-0",
+    "aiohttp",  # vcrpy requirements.
```

Comment:
we prob. can remove the vcrpy requirements for now.

The lines
```
 "pytest-assume",  # sql counter check
    "decorator",  # sql counter check
```
already exist on LL 55-66

Reply:
Done.